cypress/full/buttons_page_spec.ts

```diff
@@ -0,0 +1,213 @@
+import { ButtonsPo } from '../support/buttons.po';
+
+describe('Buttons page test suite', () => {
+  const buttons = new ButtonsPo();
+
+  beforeEach(() => buttons.navigateTo());
+
+  describe('Checkbox', () => {
+    const checkboxDemo = buttons.exampleDemosArr.checkbox;
+
+    it('example contains 3 btns and output window,btns can be active or vs. as default - picked Middle btn', () => {
+      buttons.isButtonsVisibleInGroup(checkboxDemo, 'Left');
+      buttons.isButtonsVisibleInGroup(checkboxDemo, 'Middle');
+      buttons.isButtonsVisibleInGroup(checkboxDemo, 'Right');
+      buttons.isBtnCheckboxSelected(checkboxDemo, 1, true);
+      buttons.isBtnCheckboxSelected(checkboxDemo, 0, false);
+      buttons.isBtnCheckboxSelected(checkboxDemo, 2, false);
+    });
+
+    it('when user clicks on the Left button it became active and after user clicks again - inactive', () => {
+      buttons.isButtonsVisibleInGroup(checkboxDemo, 'Left');
+      buttons.isBtnCheckboxSelected(checkboxDemo, 0, false);
+      buttons.clickByText(checkboxDemo, 'Left');
+      buttons.isBtnCheckboxSelected(checkboxDemo, 0, true);
+      buttons.clickByText(checkboxDemo, 'Left');
+      buttons.isBtnCheckboxSelected(checkboxDemo, 0, false);
+    });
+
+    it('when user clicks on the Middle button it became inactive and after user clicks again - active', () => {
+      buttons.isButtonsVisibleInGroup(checkboxDemo, 'Middle');
+      buttons.isBtnCheckboxSelected(checkboxDemo, 1, true);
+      buttons.clickByText(checkboxDemo, 'Middle');
+      buttons.isBtnCheckboxSelected(checkboxDemo, 1, false);
+      buttons.clickByText(checkboxDemo, 'Middle');
+      buttons.isBtnCheckboxSelected(checkboxDemo, 1, true);
+    });
+
+    it('when user clicks on the Right button it became active and after user clicks again - inactive', () => {
+      buttons.isButtonsVisibleInGroup(checkboxDemo, 'Right');
+      buttons.isBtnCheckboxSelected(checkboxDemo, 2, false);
+      buttons.clickByText(checkboxDemo, 'Right');
+      buttons.isBtnCheckboxSelected(checkboxDemo, 2, true);
+      buttons.clickByText(checkboxDemo, 'Right');
+      buttons.isBtnCheckboxSelected(checkboxDemo, 2, false);
+    });
+  });
+
+  describe('Custom checkbox value', () => {
+    const customCheckboxVal = buttons.exampleDemosArr.customCheckboxVal;
+
+    it('example contains btn and output window', () => {
+      buttons.isButtonVisible(customCheckboxVal, 'Single Toggle');
+      buttons.isPreviewExist(customCheckboxVal, '1');
+    });
+
+    it('output value , can be changed by click on button', () => {
```

Comment:
when user clicks on "Single Toggle", button become inactive and custom checkbox value should be "0"